tcms/static/js/testrun_actions.js

```diff
@@ -244,63 +252,42 @@ Nitrate.TestRuns.AssignCase.on_load = function() {
   });
 };
 
+function updateExecutionStatus(executionId, caseId, executionStatusPk) {
 
-function updateRunStatus(object_pk, value, callback) {
-  jQ.ajax({
-    'url': '/runs/case-run-update-status/',
-    'type': 'POST',
-    'data': {'object_pk': object_pk, 'status_id': value },
-    'success': function (data, textStatus, jqXHR) {
-      callback();
-    },
-    'error': function (jqXHR, textStatus, errorThrown) {
-      json_failure(jqXHR);
-    }
-  });
-}
+  const container = $('.update_form').parents().eq(3)
+  const parent = container.parent()
+  const title = parent.prev()
+  const link = title.find('.expandable')[0];
```

Comment:
You have the container in the calling context, pass it as argument instead of searching globally on the page. And then get the values of the other elements from the current container.